src/components/i18n.js

```diff
@@ -0,0 +1,43 @@
+/**
+ * Component for localization functions.
```

Comment:
This comment belongs to line 27 I guess?

Reply:
Yes, i will move it down